ci-operator/config/openshift/telemeter/openshift-telemeter-master.yaml

```diff
@@ -41,11 +41,11 @@ tests:
   container:
     from: src
 - as: unit
-  commands: make lint test-unit
+  commands: make test-unit
   container:
     from: src
-- as: format
-  commands: make test-format
+- as: lint-and-format
```

Comment:
(nit) I would call it `lint` for simplicity (e.g. formatting is a subset of linting IMHO).

Reply:
Agree, I'm also more keen on the short naming, let me update it ;)

Reply:
Agree, I'm also more keen on the short naming, let me update it ;)